### DISCUSSION TITLE
Alternative Sponsorship Opportunities

### DISCUSSION THREAD

**Discussion by jessejamesjohnston** [FreshRSS/FreshRSS#4997] [General]
FreshRSS has been an incredibly powerful tool for me over the past few years, and I'd very much like to configure a recurring sponsorship payment to your team in return, however I see you are currently only accepting donations via liberapay.

Is there any interest in your team possibly also accepting sponsors via a more common platform, such as Open Collective?

**Comment by Alkarex** [comment]
Thanks for reaching out and for the kind message!
I am not familiar with https://opencollective.com but we could have a look.
Do you know it @marienfressinaud ?

**Reply by marienfressinaud** [reply]
I only know it by name, I've never used it

**Reply by jessejamesjohnston** [reply]
It seems pretty popular in the FOSS community (e.g., I already donate monthly to linuxserver.io, Sonarr, and Radarr devs there).

**Comment by marienfressinaud** [comment]
Related to https://github.com/FreshRSS/FreshRSS/issues/1694 (the discussion that led to the creation of the Liberapay account)
